cloudbuild.yaml

```diff
@@ -4,4 +4,6 @@ steps:
 - name: 'gcr.io/cloud-builders/docker'
   args: ['push', 'gcr.io/flowlink-project/github.com/flowlink/${_DOCKERTAG}:${SHORT_SHA}']
 - name: 'gcr.io/cloud-builders/gcloud'
-  args: ['beta', 'run', 'deploy', '${_CRNAME}', '--image', 'gcr.io/flowlink-project/github.com/flowlink/${_DOCKERTAG}:${SHORT_SHA}', '--region', 'us-central1', '--platform', 'managed', '--allow-unauthenticated']
\ No newline at end of file
+  args: ['beta', 'run', 'deploy', '${_CRNAME}', '--image', 'gcr.io/flowlink-project/github.com/flowlink/${_DOCKERTAG}:${SHORT_SHA}', '--region', 'us-central1', '--platform', 'managed', '--allow-unauthenticated']
```

Comment:
Very clever application of the update-traffic thing, and there are ENV vars here I don't recognize!  What's _DOCKERTAG and _CRNAME? Where are they defined? Love to learn new things 😺

Reply:
Lol, I don't know if I copied it right but I got this from the cloud build update you did for the systum integration. I saw the `_DOCKERTAG` on line 3 and assumed the "_" environment variables are from cloud build

Reply:
Yeah, those variables come from Cloud Build. Applause all around!